src/lib/numbers.js

```diff
@@ -0,0 +1,67 @@
+function add (a, b) {
+  const result = a + b;
+  return result;
+}
+
+function subtract (a, b) {
+  const result = a - b;
+  return result;
+}
+
+function multiply (a, b) {
+const result = a * b;
+return result;  
+}
+
+function divide (a, b) {
+  const result = a / b;
+  return result;
+}
+
+function power (a, b) {
+  const result = a ** b;
+  return result;
+}
+
+function round (a) {
+    const result = Math.round(a);
+   return result;
+}
+
+function roundUp (a) {
+  const result = Math.ceil(a);
+  return result;
+}
+
+function roundDown (a) {
+  const result = Math.floor(a);
+  return result;
+}
+
+function absolute (a) {
+  const result = Math.abs(a);
+  return result;
+}
+
+function quotient (a, b) {
+return parseInt(a/b); 
+
+}
+
+function remainder (a, b) {
+  return parseInt(a%b);
+}
```

Comment:
You should check for formatting by running prettier across your whole project, it will fix things like this indentation and spacing inconsistency here for you:
```
function quotient (a, b) {
  return parseInt(a / b); 
}

function remainder (a, b) {
  return parseInt(a % b);
}
```